src/styles/header.scss

```diff
@@ -0,0 +1,79 @@
+.main {
+  background-color: white;
+}
+
+.header {
+  width: 100%;
+  background-color: #fff;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  z-index: 1;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+  margin-left: auto;
+
+  display: flex;
+  align-items: center;
+}
+
+.nav__link {
+  margin: 0 10px;
+  height: 60px;
+
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+
+  position: relative;
+}
+
+.style {
+  text-decoration: none;
+  color: $nav-link-color;
+  padding: 23px 0;
+  transition: 300ms;
```

Comment:
It's better to specify to what properties you need to apply the transition.

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/c7e54df5-c519-46c2-894b-184146a4abeb)